mq/esworker_sqs.py

```diff
@@ -157,7 +156,7 @@ def esConnect():
 
 class taskConsumer(object):
 
-    def __init__(self, mqConnection, taskQueue, esConnection):
+    def __init__(self, mqConnection, taskQueue, esConnection, options):
```

Comment:
This will throw a syntax error, we don't need the options parameter here.

Reply:
Are these workers not covered in the test suite?